index.html

```diff
@@ -50,7 +50,8 @@
 </head>
 
 <body>
-  <section id='abstract' data-include="spec/01-abstract.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/00-abstract.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/01-processing-model.md" data-include-format='markdown'></section>  
```

Comment:
this should be moved to after SOTD

Reply:
I did not really change any order. Which order are you proposing?